src/shared/components/FlexAlignedCheckbox.tsx

```diff
@@ -14,14 +14,14 @@ export interface ICheckboxProps {
 export default class FlexAlignedCheckbox extends React.Component<ICheckboxProps, {}> {
     render() {
         return (
-            <div className="checkbox" style={Object.assign({display:"flex", alignItems:"center"}, this.props.style)}>
+            <div style={Object.assign({display:"flex", alignItems:"center"}, this.props.style)}>
                 <label>
                     <input
                         type="checkbox"
                         value={this.props.value}
                         checked={this.props.checked}
                         onClick={this.props.onClick}
-                        style={{marginTop:1}}
+                        style={{marginTop:1, marginRight:5}}
```

Comment:
lets apply all these styles (above too) in css classes.  We shouldn't use inline styles in library code.  If it appears many times on a page, inline css is unperformant.